src/smk/viewer-leaflet/viewer-leaflet.js

```diff
@@ -226,19 +228,19 @@ include.module( 'viewer-leaflet', [ 'viewer', 'leaflet', 'layer-leaflet', /*'fea
             } )
         }
 
-        this.currentBasemap = this.createBasemapLayer( basemapId );
+        this.currentBasemap = this.createBasemapLayer( esriApiKey, basemapId );
 
         this.map.addLayer( this.currentBasemap[ 0 ] );
-        this.currentBasemap[ 0 ].bringToBack();
 
         for ( var i = 1; i < this.currentBasemap.length; i += 1 )
             this.map.addLayer( this.currentBasemap[ i ] );
 
         this.changedBaseMap( { baseMap: basemapId } )
     }
 
-    ViewerLeaflet.prototype.createBasemapLayer = function ( basemapId ) {
-        return this.basemap[ basemapId ].create( basemapId )
+    ViewerLeaflet.prototype.createBasemapLayer = function ( esriApiKey, basemapId ) {
+        const basemap = this.basemap[basemapId];
+        return basemap.create( esriApiKey, basemap.apiId || basemap.id )
```

Comment:
Is there a reason we need both of these options (basemap.apiId and basemap.id)?

Reply:
We still need id for StamenTonerLight. The value of this.basemap is defined in viewer.js, and in this review the apiId is added for all basemaps except StamenTonerLight. The id is intrinsic (which I learned while working on this :) ) and takes the value of the key, 'StamenTonerLight'.

Reply:
To satisfy my curiosity...what is an intrinsic id? Is this something unique to Vue? In this case, basemap.id always seems to be undefined for me, even in the `createBasemapTiled()` function.

Reply:
My apologies ... I'd earlier come to an incorrect conclusion. I also see that basemap.id is always undefined. I accepted the "intrinsic id" explanation because this change always worked for StamenTonerLight, but now I notice that code previously used "basemapId" and I replaced it with "basemap.id". But the reason it works is that the "id" argument in createBasemapTiled(id) is not used; in that case we create an L.tileLayer with the url and an object of other parameters.

In the next change set I'll restore "basemap.id" as "basemapId".